web/client/components/mapcontrols/search/__tests__/SearchBar-test.jsx

```diff
@@ -131,7 +151,7 @@ describe("test the SearchBar", () => {
         TestUtils.Simulate.change(input);
         TestUtils.Simulate.keyDown(input, {key: "Enter", keyCode: 13, which: 13});
         expect(spy.calls.length).toEqual(1);
-        expect(spy).toHaveBeenCalledWith('test', searchOptions, 23);
+        expect(spy).toHaveBeenCalledWith('test', searchOptions, 15);
```

Comment:
this will be the sum based on the function you wrote

Reply:
yes it is